sns/parity_math.c

```diff
@@ -96,20 +96,41 @@ static void isal_recover(struct m0_parity_math *math,
 			 enum m0_parity_linsys_algo algo);
 #endif /* ISAL_ENCODE_ENABLED */
 
+#if ISAL_ENCODE_ENABLED
+/**
+ * This is wrapper function for Intel ISA API ec_encode_data_update().
+ * @param[out] dest_buf - Array of pointers to coded output buffers
```

Comment:
s
/@param[out] dest_buf - Array of pointers to coded output buffers
/@param[out] dest_buf - Array of coded output buffers

I think this is array of "struct m0_buf", is it?

Reply:
Yes. It is array of `struct m0_buf`.